trillian/integration/hammer.go

```diff
@@ -709,8 +732,10 @@ func HammerCTLog(cfg HammerConfig) error {
 		}
 	}(ticker.C)
 
+	var nextOp ctfe.EntrypointName
 	for count := uint64(1); count < cfg.Operations; count++ {
-		if err := s.retryOneOp(ctx); err != nil {
+		var err error
+		if nextOp, err = s.retryOneOp(ctx, nextOp); err != nil {
```

Comment:
Could `nextOp` not be part of `hammerState` and be "chosen" by `chooseOp()` next time around? (I think that's essentially what you're doing here, but by pushing it back out to the caller to keep that state?)

That'd make it easy e.g. to keep a stack of these so a dependency chains like GetEntries ->Need(AddChain) -> Need(GetSTH) could be handled in order without unblocking and then waiting for the dice to hit the previously blocked op. Maybe?

Reply:
Good idea, done.